### PR TITLE
Refactor translations to adhere current app structure

### PR DESCRIPTION
**Pull request by DimalChandrasiri**
## Purpose
This PR will refactor the i18n translations to adhere to the current apps structure. Hence the `userPortal` namespace will be changed to `myAccount`. 

The `adminPortal` and `devPortal` namespaces will be moved to a common namespace named `console` and the `adminPortal` namespace will be refactored to `manage`. The `devPortal` namespace will be refactored to `develop`.

fixes https://github.com/wso2/product-is/issues/9532

## Goals
This will update the translations to adhere to the new portal structure.

### CODE REVIEW

File: modules/i18n/src/constants.ts [hunk 1]
```diff
@@ -46,29 +46,13 @@ export class I18nModuleConstants {
...
-    public static readonly ADMIN_PORTAL_NAMESPACE: string = "adminPortal";
-
     /**
      * User portal namespace.
      * @constant
      * @type {string}
      * @default
      */
-    public static readonly USER_PORTAL_NAMESPACE: string = "userPortal";
+    public static readonly MYACCOUNT_NAMESPACE: string = "myAccount";
```
Comment (brionmario) [inline modules/i18n/src/constants.ts:55]: Shall we rename this to `MY_ACCOUNT_NAMESPACE`. WDTY?

File: modules/react-components/src/components/input/url-input.tsx [hunk 2]
```diff
@@ -328,7 +328,7 @@ export const URLInput: FunctionComponent<URLInputPropsInterface> = (
             return (
                 <LabelWithPopup
                     className="cors-details-popup"
-                    popupHeader={ t("devPortal:components.URLInput.withLabel.positive.header") }
+                    popupHeader={ t("console:develop.features.URLInput.withLabel.positive.header") }
```
Comment (brionmario) [inline modules/react-components/src/components/input/url-input.tsx:331]: React Components module should not depend on the locale bundles of the app. Ideally, URLInput component should have a default english strings as `defaultProps` and users should be able to pass the translated string to the component. Shall we add a TODO and a link to an issue to get this fixed. Need to do the same for `TransferList` component as well.

File: modules/react-components/src/components/transfer/transfer-list.tsx [hunk 3]
```diff
@@ -101,7 +101,7 @@ export const TransferList: FunctionComponent<TransferListPropsInterface> = (
                             subtitle={ [
                                 emptyPlaceholderContent
                                     ? emptyPlaceholderContent
-                                    : t("devPortal:components.transferList.list.emptyPlaceholders.default")
+                                    : t("console:manage.features.transferList.list.emptyPlaceholders.default")
```
Comment (brionmario) [inline modules/react-components/src/components/transfer/transfer-list.tsx:104]: Add a TODO and an issue to remove the usage of translation namespaces inside `React Components` module.

**Review by brionmario — APPROVED** [review]

File: modules/i18n/src/constants.ts (hunk 1 above)
Comment (DimalChandrasiri) [reply modules/i18n/src/constants.ts]: +1 and fixed with commit 76d6c86

File: modules/react-components/src/components/input/url-input.tsx (hunk 2 above)
Comment (DimalChandrasiri) [reply modules/react-components/src/components/input/url-input.tsx:336]: fixed with commit 76d6c86. This issue will track the mentioned issue. https://github.com/wso2/product-is/issues/10693

File: modules/react-components/src/components/transfer/transfer-list.tsx (hunk 3 above)
Comment (DimalChandrasiri) [reply modules/react-components/src/components/transfer/transfer-list.tsx:109]: Fixed with commit 76d6c86. This issue will track the mentioned issue. https://github.com/wso2/product-is/issues/10693

**Pull request merged**